build/common.go

```diff
@@ -85,10 +85,13 @@ func findRunningPIDs(exe string) []int {
 	return pids
 }
 
-func killAllPIDs(pids []int) error {
+func killAllPIDs(exeName string) error {
```

Comment:
should likely update the function name too ;)

Reply:
@ryantxu Why rename it? What should the new name be eventually?